### DISCUSSION TITLE
keep response validation message internal

### DISCUSSION THREAD

**Discussion by bluebrown** [ogen-go/ogen#1284] [Q&A] [answered]
Hi, I enabled  `server/response/validation`, but I dont want to send the validation error out to the client. I want to log it and return a more generic message. 

I couldnt find out how to do it. 

Also, do you think this is a good default setting, to send them out? I always felt this is leaking too much info. Potentially, it can help attackers to probe out your api behavior.

**Comment by ghost** [comment, marked as answer]
You can override default error handler via `WithErrorHandler` to avoid error disclosure to the client. 

The error returned by validator should be a typed error, I guess. Feel free to fill an issue.

Feature `server/response/validation` is supposed to use in debug environment, so I think it is fine.
